lib/src/frame.dart

```diff
@@ -56,6 +80,42 @@ final _firefoxSafariFrame = RegExp(r'^'
     // empty in Safari if it's unknown.
     r'$');
 
+// With names:
+//
+// g@http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// f@http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// main@http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Without names:
+//
+// @http://localhost:8080/test.wasm:wasm-function[796]:0x143b4
+// @http://localhost:8080/test.wasm:wasm-function[795]:0x143a8
+// @http://localhost:8080/test.wasm:wasm-function[792]:0x14390
+//
+// Matches named groups:
+//
+// - "member": Function name, may be empty: `g`.
+// - "uri": `http://localhost:8080/test.wasm`.
+// - "index": `796`.
+// - "offset": (in hex) `143b4`.
+final _firefoxWasmFrame = RegExp(r'^(?<member>.*)@(?:(?<uri>.*):wasm-function'
```

Comment:
Consider making it `(?<member>.*?)@` and `(?<uri>.*?):wasm-...`.


Using a non-greedy quantifier should ensure (or at least help) that the `.*?` match stops when it first finds `:wasm-function`, and doesn't let `.*` match all the way to the end, each step adding a backtrack stack frame, and then have to lots of backtracks to get back to where `:wasm-function` occurred.

Even better, use `(?<member>[^@]*?)@`, to avoid needing any backtrack stack at all, even when finding the first `@`. The RegExp doesn't know that there couldn't be a `@` later too.

(TL;DR: Using `.*?` can be more efficient when searching for something specific right afterwards, something which `.*` can also match. Unless the RegExp compiler can optimize it, `.*?` will use fewer backtrack stack frames. It's even better if the qualifier and continuation are disjoint, so the RegExp knows it can only ever go one way.)